tests/cli/usage.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { describe, expect, it, jest } from '@jest/globals';
```

Comment:
can we use the globals that don't have to be imported? i know "magic" values aren't always the best, but it seems to be the common convention when using `jest`.

I think you can `npm install -D @types/jest` and then add `"jest"` to the `types` array in `tsconfig.json`.